test/e2e/e2e_suite_test.go

```diff
@@ -47,6 +48,25 @@ const (
 	VsphereStoragePolicy = "VSPHERE_STORAGE_POLICY"
 )
 
+const (
+	// LegacyTestMode identify tests run with CAPV in legacy mode (using govimomi to access vCenter).
+	LegacyTestMode string = "legacy"
+
+	// SupervisorTestMode identify tests run with CAPV in supervisor mode (delegating to vm-operator all the interaction with vCenter).
+	SupervisorTestMode string = "supervisor"
+)
+
+const (
+	// VMCTestTarget identify tests targeting VMC infrastructure used for CAPV CI.
+	VMCTestTarget string = "vmc"
+
+	// BYOTestTarget identify tests targeting a user provided vCenter instance.
+	BYOTestTarget string = "byo"
```

Comment:
Wondering if we should simply call this VCenterTestTarget (you also have to BYO VMC if you want to run the tests)